srcpkgs/ngx_headers_more/template

```diff
@@ -0,0 +1,39 @@
+# Template file for 'ngx_headers_more'
+pkgname=ngx_headers_more
```

Comment:
Why did you choose to name this package ngx_headers_more? Other distributions use nginx-mod-headers-more or something similar.

Reply:
I took the name directly from the readme: https://github.com/openresty/headers-more-nginx-module#name. We could also take the name "headers-more-nginx-module" since that's that what the repo is called, but since the README explicitly lists a different name, I thought that would be the appropriate name to choose.

Reply:
In this case, I would go with nginx-headers-more or nginx-mod-headers-more, because imo it makes more sense than ngx_headers_more. I personally like prefixing modules with the base package name. Naming this package is important, because it will influence naming of all other nginx modules.

It's not a taboo to name packages the same way as other distributions. When there are reasons to name a package a specific way, it doesn't matter if the package is named similarly elsewhere or not. My point was that no other distribution I looked at went with ngx_headers_more, so I don't see a reason why we should.